setup/instance/inputs.go

```diff
@@ -0,0 +1,245 @@
+Password: The password input variable specifies the password portion of basic auth used to secure
+the Layer0 API. This value will be base64 encoded along with the username input variable to
+generate your LAYER0_AUTH_TOKEN.
+`
+
+const INPUT_AWS_SSH_KEY_PAIR_DESCRIPTION = `
+AWS SSH Key Pair: The ssh_key_pair input variable specifies the name of the 
+ssh key pair to include in EC2 instances provisioned by Layer0. This key pair must 
+already exist in the AWS account. The names of existing key pairs can be found
+in the EC2 dashboard. Note that changing this value will not effect instances 
+that have already been provisioned. 
+`
+
+const INPUT_VPC_ID_DESCRIPTION = `
+VPC ID (optional): The vpc_id input variable specifies an existing AWS VPC to provision
+the AWS resources required for Layer0. If no input is specified, a new VPC will be
+created for you. Existing VPCs must satisfy the following constraints:
+
+    - Have at least 1 public and 1 private subnet
+    - Each subnet must be tagged with ["Tier": "Private"] or ["Tier": "Public"]
+
+Note that changing this value will destroy and recreate any existing resources.
+`
+
+type ModuleInput struct {
+	Name        string
+	Description string
+	Default     interface{}
+	StaticValue interface{}
+	prompter    func(ModuleInput, interface{}) (interface{}, error)
+}
+
+func InitializeLayer0ModuleInputs(version string) {
+	if version == "" {
+		logrus.Warningf("Version not set. Using default values for 'source' and 'version' inputs")
+		return
+	}
+
+	for _, input := range Layer0ModuleInputs {
+		switch input.Name {
+		case INPUT_SOURCE:
+			input.Default = fmt.Sprintf("%s?ref=%s", LAYER0_MODULE_SOURCE, version)
+		case INPUT_VERSION:
+			input.Default = version
+		}
+	}
+}
+
+var Layer0ModuleInputs = []*ModuleInput{
+	{
+		Name:        INPUT_SOURCE,
+		Description: INPUT_SOURCE_DESCRIPTION,
+		Default:     fmt.Sprintf("%s?ref=master", LAYER0_MODULE_SOURCE),
+		prompter:    RequiredStringPrompter,
+	},
+	{
+		Name:        INPUT_VERSION,
+		Description: INPUT_VERSION_DESCRIPTION,
+		Default:     "latest",
+		prompter:    RequiredStringPrompter,
+	},
+	{
+		Name:        INPUT_AWS_ACCESS_KEY,
+		Description: INPUT_AWS_ACCESS_KEY_DESCRIPTION,
+		prompter:    RequiredStringPrompter,
+	},
+	{
+		Name:        INPUT_AWS_SECRET_KEY,
+		Description: INPUT_AWS_SECRET_KEY_DESCRIPTION,
+		prompter:    RequiredStringPrompter,
+	},
+	{
+		Name:        INPUT_AWS_REGION,
+		Description: INPUT_AWS_REGION_DESCRIPTION,
+		Default:     aws.DEFAULT_AWS_REGION,
+		prompter:    RequiredStringPrompter,
+	},
+	{
+		Name:        INPUT_AWS_SSH_KEY_PAIR,
+		Description: INPUT_AWS_SSH_KEY_PAIR_DESCRIPTION,
+		prompter:    RequiredStringPrompter,
+	},
+	{
+		Name:        INPUT_USERNAME,
+		Default:     "layer0",
+		Description: INPUT_USERNAME_DESCRIPTION,
+		prompter:    RequiredStringPrompter,
+	},
+	{
+		Name:        INPUT_PASSWORD,
+		Description: INPUT_PASSWORD_DESCRIPTION,
+		Default:     "nohaxplz",
```

Comment:
I don't think this is a good default. Not sure if we should even have a default for this param.